src/app/modules/bounce/bounce.service.ts

```diff
@@ -109,11 +122,20 @@ export const isValidSnsRequest = async (
   return isValid
 }
 
-// Writes a log message if all recipients have bounced
+/**
+ * Logs all details of lost submission.
+ * @param bounceDoc Document from the Bounce collection
+ * @param notification Notification received from SNS
+ * @param autoEmailRecipients Recipients who were emailed
+ * @param autoSmsRecipients Recipients who were SMSed
+ * @param hasDeactivated Whether the form was deactivated
+ * @returns void
+ */
 export const logCriticalBounce = (
   bounceDoc: IBounceSchema,
   notification: IEmailNotification,
   autoEmailRecipients: string[],
+  autoSmsRecipients: UserWithContactNumber[],
```

Comment:
nit: This function's arguments are getting too large, can we convert it into an object? Might require too many changes so just a nit.

for clarity, 

```ts
logCriticalBounce({
  bounceDoc: <some doc>,
  notification: <some notification>,
  ...
})
```
is much safer since the order does not matter